### ISSUE TITLE
UNDERWATER comveyor works as WET

### ISSUE THREAD

**Issue by delta-nos** [defeatedcrow/HeatAndClimateMod#235]
Hello, I discoverd that may be a bug.

my Environment (not dev version):
```
jei_1.12.2-4.16.1.301
HeatAndClimateLib_1.12.2-3.8.2
HeatAndClimateMod_1.12.2-3.8.5
```

I tested dough -> bread recipe, and egg -> boiled egg recipe like this:
![2021-12-01_12 51 56](https://user-images.githubusercontent.com/83546876/144169194-3e07a452-5492-4569-a515-ea660f0b3f75.png)

the 26 block around this conveyor block is water source, this conveyor should be returned UNDERWATER hum.
but as a result, climate checker showes WET hum, and egg -> boiled egg recipe didn't work. also dough -> bread recipe worked like this picture though jei shows this recipe works in DRY~WET (not UNDERWATER).

by the way, when I use HaC and HaCLib 3.7.1, I confirmed this behavier.

**Comment by defeatedcrow** [comment]
Thank you for your report.

This specification is disadvantageous for the player, and I think that the conveyor needs to properly bring UNDERWATER hum or TIGHT airflow, etc.

This specification is included in HaCLib and will be fixed in the next HaCLib update.

**Comment by defeatedcrow** [comment]
I thought the conveyor block should be excluded from the climate calculation, so it was fixed in 4e3bc4b.
It will be included in the next version.
